game/graphics/opengl_renderer/Shader.cpp

```diff
@@ -10,6 +10,9 @@ Shader::Shader(const std::string& shader_name, GameVersion version) : m_name(sha
   const std::string height_scale = version == GameVersion::Jak1 ? "1.0" : "0.5";
   const std::string scissor_height = version == GameVersion::Jak1 ? "448.0" : "416.0";
   const std::string scissor_adjust = "512.0 / " + scissor_height;
+
+  lg::debug("Loading shader {}", shader_name);
```

Comment:
there's some left over debug logs in this file which maybe should go?  Not sure if they will be spammy or not.  If they aren't then ignore this.

Reply:
I don't think they are very spammy, one loading per shader loaded on boot, and for a few of them debug logs are emitted if they get uniforms bound.

The default log levels are
```cpp
    lg::set_file_level(lg::level::debug);
    lg::set_stdout_level(lg::level::warn);
    lg::set_flush_level(lg::level::warn);
```